variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/services/VariantStudySummaryService.java

```diff
@@ -89,6 +90,37 @@ private ProjectionOperation projectAndFlatten() {
                 .as(VariantStudySummary.STUDY_NAME);
     }
 
+    /**
+     * the equivalent intended query is:
+     * db.files.aggregate([
+     * {$match: {date : {$gte : fromDate}}},
+     * {$group: {_id : {studyId : "$sid", studyName : "$sname"}, filesCount : {$sum : 1}}},
+     * {$project: {"studyId" : "$_id.studyId", "studyName" : "$_id.studyName", "_id" : 0, "filesCount":"$filesCount" }}
+     * ])
+     * See also the inner explanation of those 2 stages
+     *
+     * @see #matchByFromDate(Date)
+     * @see #groupAndCount
+     * @see #projectAndFlatten
+     */
+    public List<VariantStudySummary> findByFromDate(Date fromDate) {
```

Comment:
oh, even better: "findSinceDate". "from date" suggests studies from *that* date only (to me). specially in the request parameter. /studies/list?fromDate=2018-04-15 looks worse than /studies/list?sinceDate=2018-04-15. a third alternative would be "newerThan" but that may be too speech-mimicking.

Reply:
I don't like a lot "findByFromDate", but I don't think that "fromDate" is wrong. I would say that the method name for retrieving the studies in a particular date is "findByDate", not "from date". 

Other names? findFromDate, findSinceDate... if we should use "findBy" because we are doing it in all the methods, I would say "findByDateGreaterOrEqualThan" ... but I think is very verbose

Reply:
I like "findByFromDate", I think it is indicating it will retrieve results FROM the given date TO any date after that (as TO is not specified)

Reply:
To be honest I used the "findBy" because that is the idiom in Spring repositories. Even though this is a service I did it by inertia. The "from date" is as @andresfsilva says, as normally if we were to represent the interval the arguments would most probably be called `fromDate` and `toDate`.

If you guys prefer a completely different name please discuss and change the name yourselves while I'm travelling, otherwise I will merge it after arrival because there are already 2 approvals.

Reply:
I read a bit and it seems "from" is used indeed for ranges. this naming issue has already spent too much time. I'm merging and deploying so that https://github.com/EBIvariation/eva-ws/pull/89 can run travis.